### ISSUE TITLE
Dynamic configuragion

### ISSUE THREAD

**Issue by mariuszkochanowski** [Shazwazza/Smidge#75]
Do you support dynamic configuration as i described in:
https://github.com/ligershark/WebOptimizer/pull/76 
??

**Comment by Shazwazza** [comment]
Yes Smidge supports this type of thing, see: https://github.com/Shazwazza/Smidge/wiki/Declarations#view-based-declarations

There isn't tag helpers for it but it's essentially a very similar thing.

Happy to accept contributions if you would like to turn this type of declaration into tag helpers! :)
